docs/enterprise-guide/authenticating-with-saml.md

```diff
@@ -94,93 +77,102 @@ contain attributes for each User's first name, last name, and email. The asserti
+**SAML keystore path:** the absolute path to the keystore file to use for signing SAML requests.
+
+**SAML keystore password:** if it wasn't already self-evident, this is just the password for opening the keystore.
+
+**SAML keystore alias:** the alias for the key that Metabase should use for signing SAML requests.
+
+### Synchronizing group membership with your IdP
+
+This setting allows you to assign users to Metabase groups based on an attribute of your users in your IdP. Please note that this may not correlate to group functionality provided by your IdP - you may need to create a separate attribute on your users to set their Metabase group, like `metabaseGroups`.
 
-#### Configuring the group schema in your SAML provider
+#### Configuring the group schema in your IdP
 
-First, you will need to create a user attribute that you will use to indicate which Metabase groups the user should be a part of. Different SAML providers have different ways of handling this, but you will likely need to edit your user profiles. For the rest of this example, let's say that you named your attribute `metabaseGroups`.
+First, you will need to create a user attribute that you will use to indicate which Metabase groups the user should be a part of. Different IdPs have different ways of handling this, but you will likely need to edit your user profiles. For the rest of this example, let's say that you named your attribute `metabaseGroups`.
 
 Once you've created your `metabaseGroups` attribute, you will need to update it for each user you would like to be automatically added to a Metabase group. For ease of use, we recommend using the same name for the groups you would use in Metabase.
 
-After that, you will need to add an additional SAML attribute to the ones we added above. The screenshot below is for Okta, but may vary dependng on your SAML provider.
+After that, you will need to add an additional SAML attribute to the ones we added above. The screenshot below is for Okta, but may vary depending on your SAML provider.
 
 ![Group attribute](images/saml-group-attribute.png)
 
-
 #### Configuring the group schema in Metabase
 
-Once you've gotten everything set up in your SAML provider, it's just a few simple steps on the Metabase side!
+Once you've gotten everything set up in your SAML provider, there are just a few simple steps on the Metabase side.
 
-![Group schema](images/saml-group-schema.png)
+To start, make sure the toggle is set to "Enabled." Then, click Edit Mappings -> Create a Mapping. Enter in the name of one of the groups you entered as your `metabaseGroups` attribute values, then click the Add button. Next click the dropdown that appears under the `Groups` heading to select the Metabase group(s) that users with this particular `metabaseGroups` value should be added to. Then click Save.
```

Comment:
When I follow along with these instructions, that drop-down displays "No mappable groups", like this:
![image](https://user-images.githubusercontent.com/23798/69674191-99d86c80-10a4-11ea-9ccd-35fefaf0f982.png)

Questions that the docs may want to answer:

* Is that because I didn't fill in any other (valid) SAML details?
* What should I do in such a case?

I don't see any queries to the back-end in the logs, so it doesn't look like anything is looked up :man_shrugging: